.gitignore

```diff
@@ -21,3 +21,4 @@ build/
 # Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
 # should NOT be excluded as they contain compiler settings and other important
 # information for Eclipse / Flash Builder.
+playground/
```

Comment:
where is `playground` folder

Reply:
Oh, it is a folder where I put some temporary test scripts that are not included in the git. Actually I have a lot of them.